recipes/pymisc-utils/build.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 $PYTHON setup.py install
-cp misc/*.py ${PREFIX}/bin/
+find misc -name "*.py" -exec cp {} ${PREFIX}/bin/ \;
```

Comment:
What was wrong with the cp before?

Reply:
My bad, I forgot to change that back. I tested whether it caused an error I fixed with the patch.
I will change it back.

Reply:
No problem, thanks!